tutorials/forward/25_automated_coreg.py

```diff
@@ -0,0 +1,59 @@
+"""
+=========================================
+Use automated approach to co-registration
+=========================================
+
+This example shows how to use the get_mni_fiducials routine and the
+coregistration functions to perform an automated MEG-MRI co-registration.
+
+.. warning:: The quality of the co-registration depends heavily upon the
+             quality of the head shape collected during subject prepration and
+             the quality of your T1-weighted MRI. Use with caution and check
+             the co-registration error.
+"""
+
+# Author: Jon Houck <jon.houck@gmail.com>
+#
+# License: BSD-3-Clause
+
+import os.path as op
+import mne
+from mne.coreg import get_mni_fiducials, Coregistration
+from mne.io import write_fiducials, read_info
+from mne.io.constants import FIFF
+
+
+data_path = mne.datasets.sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+subject = 'sample'
+fname_fids = op.join(subjects_dir, subject, 'bem', subject + '-fiducials.fif')
+fids_mri = get_mni_fiducials(subject, subjects_dir)
+write_fiducials(fname_fids, fids_mri, coord_frame=FIFF.FIFFV_COORD_MRI)
```

Comment:
Writing something to disk should not be necessary

Reply:
I did not manage to make it work for Circle when I used `fetch_fsaverage()`.